package.json

```diff
@@ -21,8 +21,8 @@
     "./package": "./package.json"
   },
   "scripts": {
-    "test": "tap --no-esm",
```

Comment:
I'm confused, why did you remove the `--no-esm`?

Reply:
the `t.spawn` shells out to node not tap so it doesn't end up using the built in runner (and esm). At least that is my understanding

Reply:
It'd be better to put `"tap": { "esm": false }` in the package.json than have it hard-coded in the command, imo.

Reply:
Fair enough.  I was just seeing the `esm` module as unnecessary but I guess it's also not strictly required to disable it in this repository.